src/main/kotlin/org/jitsi/rtp/rtcp/RtcpPacket.kt

```diff
@@ -83,8 +84,16 @@ abstract class RtcpPacket(
                 RtcpSdesPacket.PT -> RtcpSdesPacket(buf, offset, packetLengthBytes)
                 in RtcpFbPacket.PACKET_TYPES -> RtcpFbPacket.parse(buf, offset, packetLengthBytes)
                 RtcpXrPacket.PT -> RtcpXrPacket(buf, offset, packetLengthBytes)
-                else -> TODO("unimplemented rtcp type $packetType")
+                else -> {
+                    return when (packetType) {
+                        in 192..195, in 200..213 -> UnsupportedRtcpPacket(buf, offset, packetLengthBytes)
```

Comment:
We should change `PacketExtensions.RTCP_PACKET_TYPE_RANGE` to match, and probably use it here.

Reply:
Since this isn't a continuous range, we can't define a single variable for it.  We could define something like a predicate?  Any ideas?

Reply:
`RTCP_PACKET_TYPE_RANGE = setOf(192..195, 200..213).flatten()`?

Reply:
I'm looking at making a custom, "non-continuous" range, but I think it'd probably be a hack.  Maybe a `Set` would be best.

Reply:
I don't know if we care about performance for RTCP. I guess we should.

Reply:
This is inside the outer `else` clause, why can't you use the continuous range?  It overlaps with the previous when items, but so what?